feature-utils/poly-look/src/css/colors.css

```diff
@@ -23,12 +23,13 @@
   --color-green-800: #0d532a;
 
   --color-gray-50: #f7fafc;
+  --color-gray-100: #edf2f7;
   --color-gray-300: #cbd5e0;
   --color-gray-400: #a9b6c6;
   --color-gray-500: #8d9caf;
   --color-gray-700: #525e72;
   --color-gray-800: #3e495b;
-
+ 
```

Comment:
Unneeded whitespace here.

Reply:
(We should probably look into CSS linting indeed - though incredibly out of scope here. IIRC Prettier could already do it)